verilog/formatting/tree_unwrapper.cc

```diff
@@ -1671,6 +1685,24 @@ void TreeUnwrapper::ReshapeTokenPartitions(
       // TODO(fangism): reshape for multiple assignments.
       verible::MergeConsecutiveSiblings(&partition, 0);
       VLOG(4) << "after merging 'assign':\n" << partition;
+
+      // Adjust indentation of subsequent _wrapped_ lines
```

Comment:
Given that we're here in the middle of a huge switch/case in a > 400 line function, we should probably start putting these a little bit longer code-snippets into a appropriately named function.

Reply:
Done